src/infiniteGrid.js

```diff
@@ -335,8 +336,9 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 					});
 				}
 			} else {
-				if (this.isRecycling() && this._removedContent > 0 &&
-					(ele = this._topElement || this.getTopElement())) {
+				if (this.isRecycling() && this._removedContent > 0) {
+					ele = $.isEmptyObject(this._topElement) ?
+						this.getTopElement() : this._topElement;
```

Comment:
isBottomElement and isTopElement are HTMLElement reference.

If page is leaved, HTMLElement is removed. 
When you come back page, you couldn't get previous HTMLElement reference.

and I fixed demo page and add some testcase in egjs-tc
I'm afraid that egjs-tc is private